ssh_automated_tests/core/channel.py

```diff
@@ -0,0 +1,36 @@
+from abc import ABC, abstractmethod
+
+from ssh_automated_tests.core.connection import Connection
+
+
+class Channel(ABC):
```

Comment:
I would suggest renaming to `FileChannel` as this interface declares work with files. And if the advice will be applied, rename `open_file` also.